cachi2/core/package_managers/bundler/parser.py

```diff
@@ -133,6 +148,20 @@ def validate_subpath(self) -> Self:
 
         return self
 
+    @cached_property
+    def purl(self) -> str:
+        """Get PURL for this dependency."""
+        vcs_url = get_repo_id(self.root.path).as_vcs_url_qualifier()
+        qualifiers = {"vcs_url": vcs_url}
```

Comment:
nitpick: you can use the dict directly on the call below (note: you're doing exactly that in the next commit in this series ;) ).